app/services/xccdf_tailoring_file.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# A class representing an XCCDF Tailoring File
+class XccdfTailoringFile
+  extend Forwardable
+
+  XCCDF = 'xccdf'
+
+  def initialize(profile:, rule_ref_ids: [], set_values: {})
+    @profile = profile
+    @rule_ref_ids = rule_ref_ids
```

Comment:
It looks like nothing at all is using these 2 vars yet, so this can only produce just the blueprint of a tailoring file, is that right?

Reply:
That's correct. This is for tailoring in the future and is currently not used.